server/models/Profile.js

```diff
@@ -0,0 +1,40 @@
+import mongoose from 'mongoose'
+
+const profileSchema = mongoose.Schema({
```

Comment:
The spacing/indentation seems off, use a code formatter to ensure consistent formatting. Please see below for resources on setting it up:

VS Code: https://www.loom.com/share/bb8aaa6234ff4ce1b48306d16e268486
Atom: https://www.loom.com/share/8025d3c6bd8343e4bee1a0cb61ee1698
Sublime Text: https://www.loom.com/share/12ae38b15a774887924c94b56561861c